frontend/packages/ux-editor/src/components/Properties/DeprecatedCalculationsInfo.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Alert, Heading, Link } from '@digdir/designsystemet-react';
+import { ExternalLinkIcon } from '@studio/icons';
+import { StudioParagraph } from '@studio/components';
+import { useAppContext, useText } from '../../hooks';
+import classes from './Calculations.module.css';
+import { altinnDocsUrl, giteaEditLink } from 'app-shared/ext-urls';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import type { WindowWithRuleModel } from '@altinn/ux-editor/hooks/queries/useRuleModelQuery';
+
+export const DeprecatedCalculationsInfo = () => {
+  const t = useText();
```

Comment:
should this not be `useTranslation` as we use everywhere else in our code? 😄

Reply:
useText is used many places also. We do not have a system for it, but useText is just a typed version that we have made in Studio I think 🤔